src/models/stores/documents.ts

```diff
@@ -1,6 +1,7 @@
 import { types } from "mobx-state-tree";
 import { DocumentModel, DocumentModelType, DocumentType, LearningLogDocument, LearningLogPublication,
-        OtherDocumentType, OtherPublicationType, PersonalDocument, PersonalPublication, ProblemDocument
+        OtherDocumentType, OtherPublicationType, PersonalDocument, PersonalPublication, ProblemDocument,
+        PublicationDocument
```

Comment:
There will be conflicts with my teacher supports PR where I've (finally) renamed `PublicationDocument` to `ProblemPublication` for consistency, but I'll deal with that when it happens.